### DISCUSSION TITLE
Low Zoom Level Groundcover

### DISCUSSION THREAD

**Discussion by helloimalastair** [onthegomap/planetiler#819] [Q&A] [answered]
Hey y'all!

I've been noticing recently that most self-hosted/generated maps don't support displaying groundcover(at least I think it is groundcover) in lower zoom levels, but hosted maps like Mapbox/Google Maps support groundcover at all levels, even z0. Is this due to there not being sources of ground cover for lower zoom levels? Or that it costs too much to downscale? Or some other reason?

**Comment by msbarry** [comment, marked as answer]
Hello! There are a couple of options here. @wipfli has been working on https://github.com/wipfli/esa-worldcover-polygons and also the daylight team at meta put out https://daylightmap.org/2023/10/11/landcover.html which you could incorporate into a planetiler profile. I haven't played around with these much yet but hoping to add one as a default option in planetiler at some point.

**Reply by wipfli** [reply]
Hi! I think there are not many great open source options at the moment. If you want to do something that looks like google maps you need at least these elements:

* Landcover (trees, crops, geass, snow), probably ESA is good
* Climate zone (arid areas vs wet areas)
* Vector hillshading
* Clustering of building footprints for village and city polygons

One challenge is building all these elements in the first place. I have tried landcover and climate zone but vector hillshade and building clusters still are open. The other challenge is blending all these elements nicely togther.

Google and apple maps have done an amazing job at this and accordingly user expectations are high...

By the way apply does not have the vector hillshading and still looks very good.

In the end the difficulty really is generalizing small scale things like forests to low zoom levels. You want to have low-poly representations because of tile size but also to make the map look calmer.

**Reply by wipfli** [reply]
https://wipfli.github.io/median-filter-landcover/#map=2.08/23.52/45.46 is a climate zone dataset

**Reply by msbarry** [reply]
@wipfli Is your plan to mix climate zones with ground cover somehow?

**Reply by wipfli** [reply]
Yes, in the tile post process hook. The old google maps landcover had like 3 greens and 2 or 3 browns/oranges. We can use greens in non-arid areas and browns in arid areas...
